mlir/include/mlir/Dialect/MIOpen/Tuning/GridwiseGemmParams.h

```diff
@@ -966,6 +968,8 @@ class PopulateParamsXDL : public PopulateParamsBase {
     if ((param.gemmNPerBlock % param.gemmNPerWave) != 0)
       return failure();
 
+    // TODO Remove. Note KPerBlock and KPack are independent tuning parameters.
+    // There's no need to check if they are divide exactly
```

Comment:
Correct this needs to remove. However removing this would allow more tuning parameters to run. For i8 this is fine, however, once I turn this check off it will result in a crash in fp16 condition. I'll file a new ticket for the context.